src/headers/tomcrypt_pk.h

```diff
@@ -248,10 +248,12 @@ typedef struct {
 typedef enum ecc_signature_type_ {
    /* ASN.1 encoded, ANSI X9.62 */
    LTC_ECCSIG_ANSIX962   = 0x0,
-   /* raw R, S values */
+   /* raw R, S values - both values strictly padded to the size of the curve order*/
    LTC_ECCSIG_RFC7518    = 0x1,
+   /* raw R, S values - without strict padding check (used just for verification) */
+   LTC_ECCSIG_RFC7518_RELAXED = 0x2,
```

Comment:
if we really change it, would it probably make more sense to make this enum (at least partially) a bitfield and relaxed is a flag (e.g. `0x80`) where there's currently only a relaxed implementation of RFC 7518?